client/gutenberg/extensions/seo/panel.jsx

```diff
@@ -0,0 +1,61 @@
+/** @format */
```

Comment:
We can remove these, Prettier now does all the files. 🙌

Reply:
Ugh, leftover. Thanks, good spot!